docs/dev/tools/cli.md

```diff
@@ -868,7 +868,7 @@ viam module upload --version=1.0.0 --platform=darwin/arm64 packaged-module.tar.g
 | -------------- | ----------- | -------------------- |
 | `generate` | Auto-generate stub files for a new module by following prompts. | - |
 | `create` | Generate new metadata for a custom module on your local filesystem and register the metadata with the Viam registry. Uploading the code that powers the module is a separate step. | - |
-| `update` | Update an existing custom module on your local filesystem with recent changes to the [`meta.json` file](/operate/get-started/other-hardware/#metajson-reference). The `upload` command automatically runs `update` for you; you do not need to explicitly run `update` if you are also running `upload`. | - |
+| `update` | Update your module's metadata and documentation in the Viam registry based on changes to the [`meta.json` file](/operate/get-started/other-hardware/#metajson-reference) and <file>README.md</file> file. Note that `upload` automatically performs this update step. | - |
```

Comment:
```suggestion
| `update` | Update your module's metadata and documentation in the Viam registry based on changes to [<file>meta.json</file>](/operate/get-started/other-hardware/#metajson-reference) and <file>README.md</file>. Note that `upload` automatically performs this update step. | - |
```
[nit] we can do this in fewer words! and unless I'm mistaken, `meta.json` ought to use `<file>` markup instead of standard backticks here.

Reply:
Ah you're right! I think "the" should stay in though...at least that's how I would say it:
```suggestion
| `update` | Update your module's metadata and documentation in the Viam registry based on changes to the [<file>meta.json</file>](/operate/get-started/other-hardware/#metajson-reference) and <file>README.md</file>. Note that `upload` automatically performs this update step. | - |
```